app/controllers/point_detail_comments_controller.rb

```diff
@@ -0,0 +1,51 @@
+class PointDetailCommentsController < ApplicationController
+  before_action :set_point_detail_comment, only: %i[edit update]
+  before_action :authorize
+
+  def index
+    comments = comments_by_principle_user_id(params[:principle_id].to_i, params[:user_id].to_i)
+    @point_detail_comments = PointDetailCommentDecorator.decorate_collection(comments)
+    @point_detail_comment = PointDetailComment.new
+    render layout: false
+  end
+
+  def create
+    create_comment = CreatePointDetailComment.new(params[:evaluation_id], params[:principle_id],
+                                                  params[:user_id], current_user)
+    point_detail_comment = create_comment.call(update_params[:comment])
+    return head :forbidden unless point_detail_comment&.valid?
+
+    @point_detail_comment = point_detail_comment.decorate
+  end
+
+  def update
+    @point_detail_comment.update update_params
+  end
+
+  def edit; end
+
+  private
+
+  def set_point_detail_comment
+    @point_detail_comment = PointDetailComment.find(params[:id]).decorate
```

Comment:
I am not really know, that is a good idea to decorate here or not.
I see this way we save some line, but if we don't view this method, then we don't really know that the instance is decorated.